### PR TITLE
Fixed Redundant Empty Bookmark Screen

### LINKED ISSUE #338: Redundant empty bookmarks screen
When there are no bookmarks, the Bookmarks screen just shows "No bookmarked session yet !!". 

It's a really bad idea to do this. You're letting the user press the Bookmarks option on the NavigationDrawer, and lead them to nothing, which is awful. And it's wasting time.

We don't need to lead the user to an entire blank screen with nothing on it to tell him that no bookmarks exist.

### PR DESCRIPTION
**Pull request by shivenmian**
Fixes #338. 

Added a simple check once the user presses the Bookmarks option on NavDrawer. If no bookmarks exist, it shows up a prompt rather than lead him to an empty screen, thus saving time of the user. So the BookmarksFragment gets loaded only when some sessions are bookmarked.

Also, if we un-bookmark a session and the Bookmarks list becomes empty again, it shows up the dialog on pressing the back button on that Session Detail, and clicking on OK loads the TracksFragment back again.

@championswimmer @mananwason

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/fragments/BookmarksFragment.java [hunk 1]
```diff
@@ -1,9 +1,14 @@
 package org.fossasia.openevent.fragments;
 
+import android.content.DialogInterface;
 import android.content.Intent;
 import android.os.Bundle;
 import android.support.annotation.Nullable;
 import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v7.app.ActionBarActivity;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/fragments/BookmarksFragment.java]: Codacy Issue found: [Avoid unused imports such as 'android.support.v7.app.ActionBarActivity'](https://www.codacy.com/app/triplez/open-event-android/file/2406628744/issues/source?bid=3233850&fileBranchId=3233898#l10)

File: app/src/main/java/org/fossasia/openevent/fragments/BookmarksFragment.java [hunk 2]
```diff
@@ -15,13 +20,15 @@
 import android.widget.TextView;
 
 import org.fossasia.openevent.R;
+import org.fossasia.openevent.activities.MainActivity;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/fragments/BookmarksFragment.java]: Codacy Issue found: [Avoid unused imports such as 'org.fossasia.openevent.activities.MainActivity'](https://www.codacy.com/app/triplez/open-event-android/file/2406628744/issues/source?bid=3233850&fileBranchId=3233898#l23)

File: app/src/main/java/org/fossasia/openevent/fragments/BookmarksFragment.java [hunk 3]
```diff
@@ -15,13 +20,15 @@
 import android.widget.TextView;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/fragments/BookmarksFragment.java]: Codacy Issue found: [Avoid unused imports such as 'android.widget.TextView'](https://www.codacy.com/app/triplez/open-event-android/file/2406628744/issues/source?bid=3233850&fileBranchId=3233898#l20)

File: app/src/main/java/org/fossasia/openevent/fragments/BookmarksFragment.java [hunk 4]
```diff
@@ -1,9 +1,14 @@
 package org.fossasia.openevent.fragments;
 
+import android.content.DialogInterface;
 import android.content.Intent;
 import android.os.Bundle;
 import android.support.annotation.Nullable;
 import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/fragments/BookmarksFragment.java]: Codacy Issue found: [Avoid unused imports such as 'android.support.v4.app.FragmentManager'](https://www.codacy.com/app/triplez/open-event-android/file/2406628744/issues/source?bid=3233850&fileBranchId=3233898#l8)

File: app/src/main/java/org/fossasia/openevent/activities/MainActivity.java [hunk 5]
```diff
@@ -51,6 +51,8 @@
 import org.fossasia.openevent.utils.SmoothActionBarDrawerToggle;
 import org.fossasia.openevent.widget.DialogFactory;
 
+import java.util.ArrayList;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/activities/MainActivity.java]: Codacy Issue found: [Avoid unused imports such as 'java.util.ArrayList'](https://www.codacy.com/app/triplez/open-event-android/file/2406628687/issues/source?bid=3233850&fileBranchId=3233898#l54)

**Comment by shivenmian** [comment]
Whoops, okay. Will refactor code and push again.

**Pull request closed without merging**